src/App.tsx

```diff
@@ -1,24 +1,182 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useEffect,
+  useState,
+  ChangeEvent,
+  FormEvent,
+} from 'react';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { ErrorPanel } from './components/ErrorPanel';
+
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+
+import { getTodos, addTodo, removeTodo } from './api/todos';
+
+const USER_ID = 6160;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [isEditing] = useState(false);
+  // const [hasChangesInTodos, setHasChangesInTodos] = useState(0);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>(todos);
+  const [selectedFilter, setSelectedFilter] = useState<Filters>(Filters.All);
+  const [todoTitleToAdd, setTodoTitleToAdd] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [idsOfLoadingTodos, setIdsOfLoadingTodos] = useState<number[]>([]);
+  const [isInputDisabled, setIsInputDisabled] = useState(false);
```

Comment:
It is better to make names more descriptive, I should understand why are you disabling input
```suggestion
  const [areTodosLoading, setAreTodosLoading] = useState(false);
```